.github/workflows/build-and-test.yml

```diff
@@ -123,12 +123,18 @@ jobs:
               run: npm install -g snarkjs
             - name: Prepare CI for Core and FFI
               run: ./scripts/prepare_ci.sh
-            - name: Run FFI tests
+            - name: Run FFI circom tests
               # TODO: Fix this custom jar thing
               run: |
                   cd mopro-ffi/
                   curl -L https://repo1.maven.org/maven2/net/java/dev/jna/jna/5.13.0/jna-5.13.0.jar -o jna-5.13.0.jar
                   CLASSPATH=jna-5.13.0.jar cargo test -- --nocapture
+            - name: Run FFI halo2 tests
+              run: |
+                  export BUILD_CONFIG_PATH="$(pwd)/config-halo2.toml"
```

Comment:
there is no `config-halo2.toml` after merging the pervious PRs
please check the test 
thank you

Reply:
Thank you, fixed